### ISSUE TITLE
starhack rules not compressed correctly

### ISSUE THREAD

**Issue by bwalding** [ben-eb/cssnano#114]
We are using bootstrap and it uses a "starhack" to support IE8. While our target is not IE8, the behaviour of the cssnano is causing rendering errors in our compressed environment (as cssnano is making rules apply to non-IE8 browsers.

Notably, the way the compressor handles \* prefixed padding properties when not all properties are prefixed with the starhack.

```
#starhack {
  *padding-top: 1px;
  padding-right: 2px;
  *padding-bottom: 1px;
  padding-left: 2px;
}
```
## Actual Output

```
#starhack{padding:1px 2px}
```
## Expected Output

```
#starhack{*padding-top: 1px;padding-right:2px;*padding-bottom:1px;padding-left:2px}
```

**Comment by bwalding** [comment]
I have a suspicion the issue might actually be in ben-eb/postcss-merge-longhand - will reraise there if appropriate.

**Comment by ben-eb** [comment]
Yep:

```sh
$ cssnano
#starhack {
  *padding-top: 1px;
  padding-right: 2px;
  *padding-bottom: 1px;
  padding-left: 2px;
}
#starhack{padding:1px 2px}
```

```sh
$ cssnano --no-mergeLonghand
#starhack {
  *padding-top: 1px;
  padding-right: 2px;
  *padding-bottom: 1px;
  padding-left: 2px;
}
#starhack{*padding-top:1px;padding-right:2px;*padding-bottom:1px;padding-left:2px}
```

**Comment by ben-eb** [comment]
Resolved in [postcss-merge-longhand 3.0](https://github.com/ben-eb/postcss-merge-longhand/commit/947185b1b19549410f943e2c24184e809746e231). I'll be bumping this once v4 is released.
